packages/ag-charts-enterprise/src/features/zoom/zoom.ts

```diff
@@ -131,14 +133,13 @@ export class Zoom extends _ModuleSupport.BaseModuleInstance implements _ModuleSu
         this.zoomManager = ctx.zoomManager;
         this.updateService = ctx.updateService;
 
-        const { PauseType } = ctx.interactionManager;
-        const interactionOpts = { bypassPause: [PauseType.ANIMATION] };
+        const dragState = InteractionState.Default & InteractionState.ZoomDrag;
         this.destroyFns.push(
-            ctx.interactionManager.addListener('dblclick', (event) => this.onDoubleClick(event), interactionOpts),
-            ctx.interactionManager.addListener('drag', (event) => this.onDrag(event), interactionOpts),
-            ctx.interactionManager.addListener('drag-end', () => this.onDragEnd(), interactionOpts),
-            ctx.interactionManager.addListener('wheel', (event) => this.onWheel(event), interactionOpts),
-            ctx.interactionManager.addListener('hover', () => this.onHover(), interactionOpts),
+            ctx.interactionManager.addListener('dblclick', (event) => this.onDoubleClick(event)),
+            ctx.interactionManager.addListener('drag', (event) => this.onDrag(event), dragState),
+            ctx.interactionManager.addListener('drag-end', () => this.onDragEnd(), dragState),
+            ctx.interactionManager.addListener('wheel', (event) => this.onWheel(event)),
+            ctx.interactionManager.addListener('hover', () => this.onHover()),
```

Comment:
Can someone clarify what's the intended behaviour of the zoom module? Do we want this listener to receive drag, wheel & hover events when an animation is in progress?

Reply:
I believe so, and any state changes should cause the animation to skip. @lsjroberts thoughts?

Reply:
Yes I'd agree on this for zoom and navigator. If the user zooms or navigates the animation should skip to the end. That is the current behaviour.

Reply:
Done. Zoom & navigation listen for some events during animation.